package.json

```
@@ -66,5 +69,20 @@
   },
   "bugs": {
     "url": "https://github.com/TimeTac/js-client-library/issues"
+  },
+  "jest": {
+    "preset": "ts-jest",
+    "testEnvironment": "node",
+    "setupFiles": [
+      "<rootDir>config.ts"
+    ],
+    "transform": {
+      ".(ts|tsx)": "ts-jest"
+    },
+    "globals": {
+      "ts-jest": {
+        "compiler": "ttypescript"
```

Looks like a typo but not sure:
```suggestion
        "compiler": "typescript"
```

---

it's not, https://typescript-tdd.github.io/ts-auto-mock/installation